buildkite/src/Constants/ContainerImages.dhall

```diff
@@ -2,7 +2,7 @@
 
 {
   toolchainBase = "codaprotocol/ci-toolchain-base:v3",
-  codaToolchain = "codaprotocol/mina-toolchain@sha256:d4ad6dec3a846e425f97de55a697de9d65327e76782337fcdcfb373f95d21dd4",
+  codaToolchain = "codaprotocol/mina-toolchain-030ed2ce08cc3d7ee57f7e1942c8278a7cef237f",
```

Comment:
Not a huge fan of SHA tagging due mostly to readability concerns tho definitely agree it's the most reliable way to ensure an image isn't changed from underneath you. Anyway, can see keeping the sha but creating and using a `latest` tag could work here.